airflow/www/utils.py

```diff
@@ -541,6 +541,13 @@ def apply(self, query, value):
         return super().apply(query, value)
 
 
+class StateAwareFilterMixin:
```

Comment:
There's nothing particular about this filter to the state column is there?

If not, it would be helpful to add this to most other columns too please.

Reply:
Nothing, I just didn't think it through, thanks @ashb .